README.md

```diff
@@ -1,6 +1,11 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+For anyone who wishes to clone this project and run it locally. You need to make .env.development for your production database and .env.test for testing the code.
```

Comment:
`.env.development` is for the development db (we'll deal with the prod db later on)

Reply:
altered. :)